curvature-collect

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
```

Comment:
Perhaps we should move all executables into a `bin` folder in order not to pollute the root directory with files